packages/types/src/types/requests/data-table-server/DataTablePreviewRequest.ts

```diff
@@ -10,5 +10,8 @@
 import { DataTable } from '../../models';
 
 export interface DataTablePreviewRequest extends DataTable {
-  runtimeParams: Record<string, unknown>;
+  /**
+   * @additionalProperties true
+   */
+  runtimeParams?: Record<string, unknown>;
```

Comment:
The ts to json schema generator puts this strictly as {} if you don't specify that it allows additional properties. 

Also, I found that it is not always present, so I set it as optional.